packages/profile-sync-controller/CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1]
+
+### Changed
+
+- `ProfileSyncController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Comment:
Looks like there are feature additions here that might make this a minor release. What are your thoughts on this?

```suggestion
## [0.2.0]

### Added

- Export `defaultState` object from `AuthenticationController` and `UserStorageController` ([#4441](https://github.com/MetaMask/core/pull/4441))

### Changed

- `AuthType`, `Env`, `Platform` are changed from const enums to enums ([#4441](https://github.com/MetaMask/core/pull/4441))

### Fixed

- `ProfileSyncController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

- Could we expand the "Fixed" entry to explain what's being fixed and any user-facing behavior changes?
- Is the removal of exports from `mockServices` user-facing? If so, it should be listed under a `### Removed` heading.

Reply:
Yeah this is fair.

Context - we are rapidly integrating into mobile and found some small issues that were not present in extension. I ended up just doing a bunch of small fixes in that PR.

To keep a consistent changelog, I'll follow the commit history I added.

NOTE - these packages are 0.x.x, so there will be some frequent changes until mobile and extension become more concrete.

Reply:
TY for comments, I've added a more descriptive changelog